audit-service/src/test/java/uk/gov/gchq/palisade/service/audit/service/ServiceName.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.audit.service;
+
+/**
+ * Enum providing type safety for service names
+ *
+ * @since 0.4.0
+ */
+public enum ServiceName {
+    TEST_SERVICE,
```

Comment:
This class should be in src rather than test. It also shouldn't contain "TEST_SERVICE"

Reply:
I don't think this should be in `src/main/java` as it will be an unused class in the jar. This class is only used at test scope.

Reply:
Done